client/src/components/overview/GalleryView.jsx

```diff
@@ -31,35 +34,11 @@ const StyledOverviewGrid = styled.div`
     'OvPicture OvPrice'
     'OvPicture OvStyle'
     'OvPicture OvForm'
+    'OvPicture blank'
+    'OvPicture blank'
     'OvDesc OvMeta'
 `;
 
-// Here is the axios request to receive the style information for a product
-const retrieveStyles = axios({
-  method: 'get',
-  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/styles',
-  headers: {
-    Authorization: API_KEY
-  }
-});
-
-// Here is the axios request to receive the product information
-const retrieveProductInfo = axios({
-  method: 'get',
-  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/',
-  headers: {
-    Authorization: API_KEY
-  }
-});
-
-// Here is the axios request to receive receive the ratings of a product
-const retrieveRatingInfo = axios({
-  method: 'get',
-  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/meta?product_id=37314',
-  headers: {
-    Authorization: API_KEY
-  }
-});
```

Comment:
I just removed these functions and put them into the overview functional component